platform/viewer/cypress/support/commands.js

```diff
@@ -469,6 +469,18 @@ Cypress.Commands.add(
   }
 );
 
+Cypress.Commands.add('openDownloadImageModal', () => {
+  // Click on More button
+  cy.get('.expandableToolMenu')
+    .as('moreBtn')
+    .click();
+
+  // Click on Download button
+  cy.get('.tooltip-inner > :nth-child(13)')
```

Comment:
Instead of `:nth-child(13)`, are we able to "get" the button we want using text?
I think it's position is more likely to change than it's label/name

Reply:
This is tricky, because the label/name can also change if user picks another language. A better/safer approach would be adding the `data-cy` selectors on the toolbar. What do you think?

Reply:
Hmmm... I'm not sure where I fall on this.
We could render `data-cy` using the english label, and have it not tied to translations.
I'm okay to keep this as `:nth-child(13)` for merge, but worth modifying the next time we touch this.

I'd prefer we implement it in `@ohif/viewer`'s toolbarRow without modifying the extension, if we do go down this route.